### ISSUE TITLE
[FEATURE] add stop/pause button

### ISSUE THREAD

**Issue by Gummibeer** [gotkrypto76/SteamAchievementManager#4]
it would be cool to stop/pause the whole loop in the main window to allow the steam process to handle all the unlocked features, be able to do something else and so on.

**Comment by gotkrypto76** [comment]
Yeah, for sure. That’d be a relatively easy thing to add, too. I’m probably going to do some digging though and check that I can’t maybe automate that cool down period so you don’t have to watch the program unlock things.
